docs/book/helper/form-element-errors.md

```diff
@@ -72,3 +72,24 @@ Method signature                                                     | Descripti
 `getAttributes() : array`                                            | Returns the attributes that will go on the message open format.
 `setTranslateMessages(bool $flag) : self`                            | Indicate whether or not element validation error messages should be translated during `render()`. Default is to translate them.
 `render(ElementInterface $element [, array $attributes = array()]) : string` | Renders validation errors for the provided `$element`. Attributes provided will be used in the `messageOpenFormat`, and merged with any provided previously via `setAttributes()`.
+
+## Usage in a laminas-mvc Application
+
+Within a [laminas-mvc](https://docs.laminas.dev/laminas-mvc/) application, the options for the view helper can be set via the application configuration.
```

Comment:
```suggestion
Within a [PSR-11 container](https://www.php-fig.org/psr/psr-11/) like the one used in [laminas-mvc](https://docs.laminas.dev/laminas-mvc/) application, the options for the view helper can be set via the container configuration.
```

Reply:
The page has no reference to `laminas-mvc` except in this new chapter.
As the new functionality has no relationship with `laminas-mvc`, I'd like not to couple to it in the documentation neither.

Maybe we can change the title to `Usage in a PSR-11 container` and then add an example that could be valid to both `laminas-mvc` and `mezzio`, wdyt?

Reply:
I am deliberately avoiding adding examples of other applications at the moment because we have no description how laminas-form can be used in Mezzio or any other application then laminas-mvc.
Yes, it's annoying, but why add something for a view helper when everything else is missing? I think it leads to more frustration than it helps when the basis is missing.

Btw. we have a project for this topic: https://github.com/orgs/laminas/projects/3

Reply:
I agree on your concern, and I trust your much wider documenting experience :+1: 

I hope one day we can only refer to `PSR`s in docs, as I think `laminas-mvc` has somehow blocked the modular usage of `laminas` ecosystem, not technically but instinctively, as every doc refers to it and it seems like a magic black box that do everything and making transition to anything else impossible.

Reply:
I totally agree with you, because I see it myself as one of the biggest problems of the documentation that the description for integration in applications is missing. And even for the own MVC system! :disappointed:

Therefore, we also need the headings and must not hide sections like this. See the navigation on the left for the page content: https://docs.laminas.dev/laminas-form/helper/form-element-errors/
With the new section there is then one more: "Usage in a laminas-mvc Application"